java/test/src/main/java/io/ray/test/RuntimeEnvTest.java

```diff
@@ -193,4 +193,38 @@ public void testZipPackageInActor() {
     testDownloadAndLoadPackage(
         "https://github.com/ray-project/test_packages/raw/main/raw_resources/java-1.0-SNAPSHOT.zip");
   }
+
+  private static boolean findClass(String className) {
+    try {
+      Class.forName(className);
+    } catch (ClassNotFoundException e) {
+      return false;
+    }
+    return true;
+  }
+
+  private static void testDownloadAndLoadPackageForTask(String url) {
```

Comment:
Can you add a test in which you input two urls and make sure both of them work?

Reply:
DONE